src/BlazingTrains/Pages/Index.razor.cs

```diff
@@ -15,11 +15,13 @@ public partial class Index
     private readonly PerSecondTimedStat _fps = InstrumentationBag.Add<PerSecondTimedStat>("SkiaSharp-OnPaintSurfaceFPS");
     private readonly ElapsedMillisecondsTimedStat _renderTime = InstrumentationBag.Add<ElapsedMillisecondsTimedStat>("GameElement-GameRender");
 
-    protected override void OnInitialized()
+    protected override async Task OnInitializedAsync()
     {
         _game = DI.ServiceLocator.GetService<IGame>();
         _interactionManager = DI.ServiceLocator.GetService<IInteractionManager>();
 
+        await _game.InitializeAsync(200, 200);
```

Comment:
Even without the async stuff, this change is worth it IMO for this line of code alone, to finally have the hosts setting column and row, rather than being hardcoded inside `PixelMapper`.